### PR TITLE
add  parameter to capture_local

### PR DESCRIPTION
**Pull request by ThinkMo**
solve #183 capture_local should include an append flag

### CODE REVIEW

**Comment by ThinkMo** [comment]
@vladimirvivien why tests failed?

File: starlark/capture_local_test.go [hunk 1]
```diff
@@ -70,6 +70,7 @@ func TestCaptureLocalFunc(t *testing.T) {
 					{starlark.String("workdir"), starlark.String("/tmp/capturecrashd")},
 					{starlark.String("file_name"), starlark.String("echo.txt")},
 					{starlark.String("desc"), starlark.String("echo command")},
+					{starlark.String("append"), starlark.True},
```
Comment (vladimirvivien) [inline starlark/capture_local_test.go:73]: Please add a new test case that actually validates whether content is appended to the file.   
You can create file at this location first `filepath.Join("/tmp/capturecrashd", "echo.txt")` then run `captureLocalFunc(newTestThreadLocal(t), nil, nil, kwargs)` then validate the content.

File: starlark/capture_local_test.go [hunk 2]
```diff
@@ -174,7 +175,7 @@ result = capture_local("echo 'Hello World!'")
 		{
 			name: "capture local with args",
 			script: `
-result = capture_local(cmd="echo 'Hello World!'", workdir="/tmp/capturecrash", file_name="echo_out.txt", desc="output command")
+result = capture_local(cmd="echo 'Hello World!'", workdir="/tmp/capturecrash", file_name="echo_out.txt", desc="output command", append=True)
```
Comment (vladimirvivien) [inline starlark/capture_local_test.go:178]: Same here. Please see comment above.

**Review by vladimirvivien — COMMENTED** [review]
@ThinkMo thinks for your contribution.  Please see comments.

**Comment by ThinkMo** [comment]
write to the result file before run test

**Review by vladimirvivien — APPROVED** [review]
This looks good.  I will approve it.
I also think you should consider open a PR to do the same for `capture` (which captures remote output of remote commands).

**Pull request merged**